inc/class-elementor.php

```diff
@@ -58,6 +59,36 @@ static function( Manager $dynamic_tags ) {
 		);
 
 		add_action( 'elementor/template-library/after_save_template', array( $this, 'fix_kit_import' ), 999, 2 );
+
+		$global_colors_experiment = Options::get_instance()->get( 'global_colors_experiment' );
+
+		if ( 'active' === $global_colors_experiment ) {
+			// This needs to be worked.
+			// add_action( 'elementor/init', array( $this, 'register_data_controllers' ), 999 );
+			$this->register_data_controllers();
+		}
+	}
+
+	/**
+	 * Register custom Elementor REST data controllers.
+	 *
+	 * @return void
+	 */
+	public function register_data_controllers() {
+		require_once ANG_PLUGIN_DIR . 'inc/elementor/globals/class-controller.php';
+		require_once ANG_PLUGIN_DIR . 'inc/elementor/globals/class-colors.php';
+
+		add_action(
+			'elementor/editor/init',
+			function() {
+				/**
+				 * Set current page id.
+				 */
+				Options::get_instance()->set( 'ang_current_page_id', get_the_ID() );
+			}
```

Comment:
@mauryaratan this is where we're saving the current page id for our REST data controller to load from active kit on page.